tests/ops/op_math/test_condition.py

```diff
@@ -203,6 +203,21 @@ def test_cond_error_unrecognized_input(self, inp, terminal_measurement):
             qml.cond(m_0, inp)()
 
 
+@pytest.mark.parametrize("op_class", [qml.PauliY, qml.Toffoli, qml.Hadamard, qml.CZ])
+def test_conditional_label(op_class):
+    """Test that the label for conditional oeprators is correct."""
+    base_op = op_class(wires=range(op_class.num_wires))
+
+    # Need to use queue because `qml.cond` doesn't return `Conditional` operators
```

Comment:
not for this PR, but maybe we should change this one day? I know it can queue more than one conditional, but it's a little strange that it returns nothing at all